geocity/apps/api/views.py

```diff
@@ -466,3 +468,91 @@ def get_queryset(self):
 SubmissionPointViewSet = submission_view_set_subset_factory("points")
 SubmissionLineViewSet = submission_view_set_subset_factory("lines")
 SubmissionPolyViewSet = submission_view_set_subset_factory("polygons")
+
+
+class AgendaViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    To get detail use /rest/agenda/:id
+    Example : /rest/agenda/7
+    Page can be given
+    Example : /rest/agenda/?page=1
+    It's possible to filter by categories
+    Example : /rest/agenda/?publics=3
+    Every filter can be cumulated
+    """
+
+    throttle_scope = "agenda"
+    serializer_class = serializers.AgendaSerializer
+    pagination_class = AgendaResultsSetPagination
+    permission_classes = [permissions.AllowAllRequesters]
+
+    def get_queryset(self):
+        """
+        This view should return a list of events for agenda with minimal information when id is not given.
+        Use used_as_api_filter to filter field by values, values (id = line of the value)
```

Comment:
Use used_as_api_filter to filter field by values. Each line of  MultipleChoiceField or SingleChoiceField is considered as an "id"